kore/src/Kore/IndexedModule/IndexedModule.hs

```diff
@@ -208,26 +224,43 @@ eraseAxiomAttributes
                     & Lens.over (Lens.mapped . Lens._3) eraseAxiomAttributes
             }
 
+eraseSyntaxAttributes ::
+    IndexedModuleSyntax patternType1 declAttributes ->
+    IndexedModuleSyntax patternType1 Attribute.Null
+eraseSyntaxAttributes
+    indexedModuleSyntax@IndexedModuleSyntax
+        { indexedModuleAliasSentences
+        , indexedModuleSymbolSentences
+        , indexedModuleImportsSyntax
+        } =
+        indexedModuleSyntax
+            { indexedModuleAliasSentences =
+                indexedModuleAliasSentences
+                    & Lens.set (Lens.mapped . Lens._1) Attribute.Null
+            , indexedModuleSymbolSentences =
+                indexedModuleSymbolSentences
+                    & Lens.set (Lens.mapped . Lens._1) Attribute.Null
+            , indexedModuleImportsSyntax =
+                indexedModuleImportsSyntax
+                    & Lens.set (Lens.mapped . Lens._1) Attribute.Null
+                    & Lens.over (Lens.mapped . Lens._3) eraseSyntaxAttributes
+            }
```

Comment:
These are all `Map`s, right? I would not use lenses with `Map`, except when you really _want_ lazy behavior. The reason for this is that `Data.Map.Strict` actually uses the same lazy `Map` type as `Data.Map.Lazy`. The only strict parts are the functions exported by the `Data.Map.Strict` module which enforce the full evaluation of the `Map`s they operate with. I believe @andreiburdusa shared something about this a while back, so feel free to jump in if you think I misunderstood anything.

Reply:
Again, I could change this if you really think it's necessary, but code of this style is already very present in this file, so it would be a very big change.